componentLibraries/defaultLibrary/Special/AeroComponents/AeroFlightRecorder.dae

```diff
@@ -0,0 +1,761 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
Is this dae model used by the calculations in any way, or is it just an example?
If it is just and example, one could question if the file should be included at all, or if it should be located someplace else.
I dont think we have any other component that comes with data files. So this would be the first.
What does @robbr48 think? 

If this file is not needed by the component it self. Maybe it would be better to place this file together with the test model. Then the test model can also use it as the chosen model. If you move it also update the markdown documentation regarding this file.
We do have other test models that come with example data files.

Reply:
If the component is depending on this file for input somehow, I think it should be there even if it is just an example. It should be possible for people to test this component out-of-the-box, without having to create custom files first. But then again, I am not even sure what this file actually does :)

Reply:
The .dae model is only used if the user chooses to have a KML output with an embedded 3D representation of the aircraft (which is the .dae file). It is not used in the cases of "Flight path" or "Ground path" (which create just polygons and lines).
In fact, the .dae is NOT used in the calculations in any case. It's just located and referenced in the output KML file, which is the one that will try to call the .dae in when opened in Google Earth. So the Hopsan component is just making sure that the user selects a .dae model and then passes on its filepath to the KML file.

I think that the best behaviour would be that this default .dae model that I included only appears if the user selects to have a 3D KML output but he/she doesn't specify a custom .dae model. The main problem is that there is no easy way to know the absolute path to the default .dae model beforehand, and the KML file needs it.

**BUT I just got an idea: The .dae model is just a static chunk of XML code. I think that it can be included in the Hopsan-component code so it automatically generates BOTH the .kml and the .dae files at the output location when it's required. I think this is the smoothest solution: then no extra file to be included with the component :)**

Reply:
While that would work, in this case I think it is better to leave it outside as its own file. Otherwise it will make the binary bigger and take up some RAM. Might even enter the CPU cache while simulating if its part of the component. Not sure how that works. Lets leave it where it is for now, it makes sense in this case.